src/main/app/src/Routes.js

```diff
@@ -1,30 +1,29 @@
-import React from 'react';
-import { Route, Switch } from 'react-router-dom';
-
-import HomePage from './pages/HomePage';
-import ScheduleActivityPage from './pages/ScheduleActivityPage';
-import BrowsePage from './pages/BrowsePage';
-import HelpPage from './pages/HelpPage';
-import AboutPage from './pages/AboutPage';
-/* Routes address bar to corresponding page components */
-class Routes extends React.Component {
-  render() {
-    return (
-      <Switch>
-        <Route exact path='/' component={HomePage} />
-        <Route exact path='/schedule-activity' component={ScheduleActivityPage} />
-        <Route exact path ='/browse' component={BrowsePage}/>
-        <Route exact path='/help' component={HelpPage} />
-        <Route exact path='/about' component={AboutPage} />
-        
-        <Route
-          render={function() {
-            return <h1>Not Found</h1>;
-          }}
-        />
-      </Switch>
-    );
-  }
-}
-
-export default Routes;
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import HomePage from './pages/HomePage';
+import ScheduleActivityPage from './pages/ScheduleActivityPage';
+import BrowsePage from './pages/BrowsePage';
+import HelpPage from './pages/HelpPage';
+import AboutPage from './pages/AboutPage';
+
+/* Routes address bar to corresponding page components */
+const Routes = ({activity, updateActivity, updateServlet, links}) => {
+    return (
+      <Switch>
+        <Route exact path='/' render={() => <HomePage activity={activity} updateActivity={updateActivity} updateServlet={updateServlet}/>}/>
+        <Route exact path='/schedule-activity' render={() => <ScheduleActivityPage activity={activity} updateActivity={updateActivity}/>}/>
+        <Route exact path ='/browse' render={() => <BrowsePage activity={activity} updateActivity={updateActivity} updateServlet={updateServlet} links={links} />}/>

```

Comment:
there is a space between exact path and =, not sure if it breaks your code.

Reply:
It doesn't but will change.

Reply:
Fixed it.